src/quantum/plugin/QuantumOptions.ts

```diff
@@ -222,7 +222,10 @@ export class QuantumOptions {
         return this.optsTarget === "electron";
     }
     public isTargetUniveral() {
-        return this.optsTarget === "universal";
+        return this.optsTarget === "universal" || this.optsTarget === "npm";

```

Comment:
Well, in general, "npm" target has the same behaviour as "universal" target. So, unless in a (only one indeed) specific case, everytime the target is tested, an "npm" target should lead to the same behaviour of an "universal" target.
This changes affect the way things are managed when `target: "npm"`, not when `target: "universal"`